app/main.py

```diff
@@ -1,3 +1,48 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __lt__(self, other: int | float | Distance) -> bool:
+        return True if self.km < other else False
```

Comment:
In this case you can do
```suggestion
        return self.km < other
```
Fix in all places